actors/builtin/market/market_balances.go

```diff
@@ -0,0 +1,115 @@
+package market
+
+import (
+	"errors"
+	"fmt"
+
+	addr "github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/specs-actors/actors/abi"
+	"github.com/filecoin-project/specs-actors/actors/abi/big"
+	"github.com/filecoin-project/specs-actors/actors/runtime/exitcode"
+	. "github.com/filecoin-project/specs-actors/actors/util"
+	"github.com/filecoin-project/specs-actors/actors/util/adt"
+	"golang.org/x/xerrors"
+)
+
+func (st *State) lockBalanceOrAbort(et, lt *adt.BalanceTable, proposal *DealProposal) (error, exitcode.ExitCode) {
```

Comment:
Perhaps just `lockBalances`, or `lockClientAndProviderBalances`.

Reply:
I think the return values deserve a comment, since it's not the usual "use the first value iff error is nil". In fact the return value is used if err is *not* nil.